src/client/app/core/core.module.ts

```diff
@@ -16,13 +17,14 @@ import { throwIfAlreadyLoaded } from './module-import-check';
   ],
   declarations: [ToggleDataSourceComponent, ToolbarComponent],
   exports: [ToggleDataSourceComponent, ToolbarComponent],
-  providers: [ToastService]
+  providers: [ NgrxDataToastService, ToastService]
 })
 export class CoreModule {
   constructor(
     @Optional()
     @SkipSelf()
-    parentModule: CoreModule
+    parentModule: CoreModule,
+    toastService: NgrxDataToastService
```

Comment:
Why here?

Reply:
It won't subscribe until the service is created. Where/when to create it? It should be at the root level (before any lazy loads). Plenty of choices after that. Creating it in the module where it is defined seemed as good a place as any.